.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: java
 
+dist: xenial
+
 matrix:
   include:
     - jdk: openjdk8
-    - jdk: oraclejdk8
```

Comment:
:'-(

Reply:
actually its necessary, on to the breach 11 here we come

Reply:
@dizzzz Xenial doesn't have Oracle JDK8 available, we could write some script to install it. As OracleJDK 8 and OpenJDK 8 have been very stable and are basically "done", I don't see them diverging at any point in the future. I suspect if our tests pass on OpenJDK 8, we will have no problem with Oracle JDK8

Reply:
@adamretter I think it is a fair assumption. Let's not invest time in the oracle jvm here.